app/src/main/res/navigation/nav_graph_favorite.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<navigation xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/tab_favorite"
+    app:startDestination="@id/favoriteListFragment">
+
+    <fragment
+        android:id="@+id/favoriteListFragment"
+        android:name="com.example.internetshop.presentation.activity.fragments.FavoriteListFragment"
+        android:label="Favorite"
+        app:popUpTo="@+id/favoriteListFragment"
+        app:popUpToInclusive="true">
+        <action
+            android:id="@+id/action_favoriteListFragment_to_productDetailsFragment2"
+            app:destination="@id/productDetailsFragment2" />
+    </fragment>
+    <fragment
+        android:id="@+id/productDetailsFragment2"
```

Comment:
Why `productDetailsFragment2`? Why not `productDetailsFragment2*32/11`? As i understand this happened because `productDetailsFragment` already has been defined in another nav graph which we don't need now. Then delete unused nav graph and use proper id in nav_graph which you use.

Reply:
fixed